gnomad_qc/v4/annotations/insilico_predictors.py

```diff
@@ -508,6 +509,91 @@ def create_phylop_grch38_ht() -> hl.Table:
     return ht
 
 
+def get_revel_for_unmatched_transcripts() -> None:
+    """
+    Create Tables with alternative REVEL scores for variants in v4.1 release.
+
+    ..note:
+        REVEL was computed using transcripts from Ensembl v64. However, in gnomAD
+        v4.0 and v4.1, we used transcript information from Ensembl v105 in addition
+        to variant information (locus and alleles combination) to ascertain variant
+        REVEL scores, We also only reported scores for MANE select or canonical
+        transcripts. Please refer to create_revel_grch38_ht() function above. This
+        means that variants within 2,414 MANE select transcripts present in gnomAD
+        v4.0/v4.1 but not present in Ensembl v64 were missing REVEL scores.
+
+        To address this, we annotated REVEL scores for variants within those 2,
+        414 transcripts using locus and allele information only (taking the maximum
+        score if a variant had multiple scores). These files contain only variants
+        from these 2,414 genes. The exomes TSV adds REVEL scores for 1,936,321 out of
+        2,284,296 (87.77%) missense variants that were previously missing REVEL
+        scores in gnomAD v4.0. The genomes TSV adds REVEL scores for 528,204 out of
+        620,799 ( 85.08%) missense variants that were previously missing REVEL scores.
```

Comment:
```suggestion
    ..note:
    
        REVEL was computed using transcripts from Ensembl v64. The gnomAD 
        v4.0 and v4.1 release HTs are annotated with transcript information from 
        Ensembl v105. Because gnomAD was only annotated with scores from
        MANE Select transcripts and due to transcript differences between Ensembl
        versions, variants within 2,414 genes in gnomAD are missing REVEL scores. 

        To address this, we annotated the variants within the 2,414 genes with the 
        maximum REVEL score found at the specific locus and allele, rather than the 
        score for the MANE Select transcript. 
        
        The exomes TSV adds REVEL scores to 1,936,321 out of 2,284,296 (87.77%)
        missense variants within the 2,414 genes. The genomes TSV adds REVEL scores
        to 528,204 out of 620,799 ( 85.08%) missense variants within the 2,414 genes.
```

Reply:
I copied from our README file, maybe I will have this version to update README?

Reply:
I'm not sure I agree with your simplified version, I wanted to emphasize that for the release HT, it was matching locus, alleles and trascript ID.

Reply:
I'm not sure about the shortened version, it lost two points: 1. it was matching locus, alleles and transcript ID for both MANE and canonical transcripts, 2. this is only for variant in MANE transcripts, matchin by locus and alleles only.